### ISSUE TITLE
AttributeError: module 'sqlalchemy' has no attribute 'Binary'

### ISSUE THREAD

**Issue by kevin-olbrich** [shosca/django-sorcery#156]
```
  File "/opt/xxxxxxx/venv/lib/python3.7/site-packages/django_sorcery/db/fields.py", line 435, in BinaryField
    type_class = sa.Binary
```

https://stackoverflow.com/questions/65851741/attributeerror-module-sqlalchemy-has-no-attribute-binary

**Comment by shosca** [comment]
Ah, sqlalchemy 1.4 is out

**Comment by shosca** [comment]
Fixed in #157 157
